packages/teams-js/test/public/chat.spec.ts

```diff
@@ -83,16 +84,34 @@ describe('chat', () => {
             message: 'someMessage',
           };
 
+          const normalizedChatRequest = {
+            members: ['someUPN'],
+            message: 'someMessage',
+          };
+
           chat.openChat(chatRequest);
 
-          const chatResponse = {
```

Comment:
>chatResponse [](http://example.com/codeflow?start=16&length=12)

It was incorrect for this to be named "response" because it was only checking that the request object that was being sent with the message was correct; it never checked the response (I've added a new test below to validate the response)